recipes/21cmFAST/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "21cmFAST" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6e88960d134e98e4719343d853c63fc3c691438b57b2863f7834f07fae9eab4f
+
+
+build:
+  number: 0
+  script: "INC=$(PREFIX)/include LIB=$(PREFIX)/lib {{ PYTHON }} -m pip install . -vv"
```

Comment:
This should be `${PREFIX}` not `$(PREFIX)`

Reply:
Ah, thanks. Done.